islandora_usage_stats.install

```diff
@@ -208,6 +208,55 @@ function islandora_usage_stats_schema() {
     ),
   );
 
+  
+  // Straight copy of islandora_usage_stats_object_ds_access_log, avoids collisions with ryserson
+  // for the time being
+  $schema['islandora_usage_stats_object_ds_download_log'] = array(
+    'description' => 'This table is used to store the usage information for Fedora datastreams.',
+    'fields' => array(
+      'id' => array(
+        'type' => 'serial',
+        'not null' => TRUE,
+      ),
+      'pid_id' => array(
+        'description' => 'PID ID of the object.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ),
+      'dsid' => array(
+        'description' => 'Datastream ID',
+        'type' => 'varchar',
+        'length' => 64,
+        'not null' => TRUE,
+      ),
+      'time' => array(
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'ip' => array(
+        'description' => 'IP of user that accessed the object',
+        'type' => 'varchar',
+        'length' => 64,
+        'not null' => TRUE,
+      ),
+      'user' => array(
```

Comment:
Making a note of this. Would require a large re-write of existing code base, and not sure how this would affect ryersons implementation (if at all).